core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -203,42 +229,127 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
     * isn't set.
     */
   @Test
-  def testNoProduceAcl {
+  def testNoProduceAclWithoutDescribeAcl {
     //Produce records
     debug("Starting to send records")
     try{
       sendRecords(numRecords, tp)
-      fail("Topic authorization exception expected")
+      fail("exception expected")
     } catch {
-      case e: TopicAuthorizationException => //expected
+      case e: TimeoutException => //expected
     }
   }
 
-  /**
+  @Test
+  def testNoProduceAclWithDescribeAcl {
```

Comment:
nit: The first "Acl" seems out of place. Should this be `testNoProduceWithDescribeAcl`? A few more like this below.